scripts/processing/05-run_ica.py

```diff
@@ -57,6 +57,7 @@ def run_ica(subject_id, tsss=None):
     ica.save(ica_name)
 
 
-parallel, run_func, _ = parallel_func(run_ica, n_jobs=N_JOBS)
+# Memory footprint: around n_jobs * 4 GB
+parallel, run_func, _ = parallel_func(run_ica, n_jobs=2)
```

Comment:
Let's just change it to `1` to be safe (users can manually increase as they wish)

Reply:
okay let me run my ICA scripts, then I change it back :)